pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[project]
+name = "python-stiebel-eltron"
+dynamic = ["version"]
+description = "Python API for interacting with the Stiebel Eltron ISG web gateway via Modbus for controlling integral ventilation units and heat pumps."
+readme = "README.md"
+authors = [
+    { name = "Manuel Stahl"},
+    { name = "'Martin Fuchs"}
+]
+license = { file = "LICENSE" }
+classifiers = [
+    "License :: OSI Approved :: MIT License",
```

Comment:
Please consider removing the following classifiers in favor of a SPDX license expression: `License :: OSI Approved :: MIT License`

See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.

Reply:
I have removed line 12